tests/rptest/perf/small_batches_test.py

```diff
@@ -0,0 +1,41 @@
+from rptest.tests.redpanda_test import RedpandaTest
+from rptest.services.openmessaging_benchmark import OpenMessagingBenchmark
+from rptest.services.openmessaging_benchmark_configs import OMBSampleConfigurations
+from rptest.services.cluster import cluster
+
+
+class SmallBatchesTest(RedpandaTest):
+    """
+    A many clients and partitions test where producers send small batches to Redpanda.
+    """
+    def __init__(self, ctx):
+        self._ctx = ctx
+        super(SmallBatchesTest,
+              self).__init__(test_context=ctx,
+                             extra_rp_conf={"aggregate_metrics": True})
+
+    @cluster(num_nodes=6)
+    def omb_test(self):
```

Comment:
When do we decide to add a new test class (as here) vs a new test within an existing class?

Reply:
It's mainly how our results aggregation script works. When inserting the results into the perf-db they use the test class name as the label for the results. Hence a new test class instead of a new method.

Reply:
I suppose I should change it to label the results as `TestClassName.MethodName` though on second thought. I'll get a PR up in vtools for that.

Reply:
See https://github.com/redpanda-data/vtools/pull/2104